tools/pelican/pelicanconf.py

```diff
@@ -51,8 +51,7 @@
 
 DEFAULT_PAGINATION = False
 
-STATIC_PATHS = ['content/pages', 'content/news', 'content/ipynb']
```

Comment:
This line I don't really understand

Reply:
I think this is right in the new version because in line 9 of the file, we set the path to `content` already. As a result, pelican is searching in `content/content/pages` etc (in the old version).